### PR TITLE
Fix typo in guide-disappearance.mdx

### PR DESCRIPTION
**Pull request by stanleyugwu**

### CODE REVIEW

File: docs/guide-disappearance.mdx [hunk 1]
```diff
@@ -69,7 +69,7 @@ test('movie title goes away', async () => {
 })
 ```
 
-## Asserting elements are not present
+## Asserting elements that are not present
```
Comment (timdeschryver) [inline docs/guide-disappearance.mdx:72]: I think "Asserting elements are not present" is more correct here, but I'm not an English native speaker 😅
Why do we need "that" here?

How I see it:
- Asserting elements are not present: you're asserting an element isn't present, e.g. you test it isn't there
- Asserting elements that are not present: you're asserting on an element that isn't present, e.g. you that that the element that isn't there isn't there (if that makes sense)

Comment (alexkrolick) [reply docs/guide-disappearance.mdx:72]: `Asserting elements are not present` or `Asserting that elements are not present` are both OK, but not this change

Comment (stanleyugwu) [reply docs/guide-disappearance.mdx:72]: OH! it just made sense when I substituted 'asserting' with 'checking' 🤣😂. 
I was thinking it means "checking elements that are not present (the result)"now I see its "checking **to know** if elements are not present".

Comment (stanleyugwu) [reply docs/guide-disappearance.mdx:72]: @alexkrolick thanks, your comment made it much clearer. And I think changing it to `Asserting that elements are not present` will be much more easy to understand.

Comment (stanleyugwu) [reply docs/guide-disappearance.mdx:72]: Thanks @timdeschryver for that explanation

**Pull request closed without merging**